subsys/mgmt/smp_shell.c

```diff
@@ -96,18 +96,17 @@ bool smp_shell_rx_byte(struct smp_shell_data *data, uint8_t byte)
 	 * The received byte is part of an mcumgr command.  Process the byte
 	 * and return true to indicate that shell should ignore it.
 	 */
-	if (data->cur + data->end < sizeof(data->mcumgr_buff) - 1) {
+	if (data->cur < sizeof(data->mcumgr_buff) - 1) {
```

Comment:
enclose arithmetic expression ()

Reply:
Could you refer to some coding style rule? Or is some tool complaining about it? Do you want it to be:
```
if (data->cur < (sizeof(data->mcumgr_buff) - 1)) {
```
?

Reply:
Yes, I thought so. There is not rule and no tool complaining about it. It just adds clarity to conditional expression.
I have searched code and it seems that we heave no consistent rule applied about this so I **will not insist** on this change. It is up to you.